src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -39,27 +40,121 @@
 
 namespace {
 
+const QString kRekordboxLibraryTable = QStringLiteral("rekordbox_library");
+const QString kRekordboxPlaylistsTable = QStringLiteral("rekordbox_playlists");
+const QString kRekordboxPlaylistTracksTable = QStringLiteral("rekordbox_playlist_tracks");
+
 const QString kPdbPath = QStringLiteral("PIONEER/rekordbox/export.pdb");
 const QString kPLaylistPathDelimiter = QStringLiteral("-->");
 const double kLongestPosition = 999999999.0;
 
-void clearTable(QSqlDatabase& database, QString tableName) {
+enum class TrackColor {
+    TrackColorPink = 1,
```

Comment:
The "TrackColor" prefix in the enum variants is now redundant

Reply:
Unfortunately getting this compiler error:
'src/library/rekordbox/rekordboxfeature.cpp:51:12: error: scoped enumeration requires a name'
'enum class : uint8_t {'

Reply:
That's not what @uklotzde meant. You should replace `TrackColor::TrackColorPink` with TrackColor::Pink`.

Reply:
Ah thank you, this makes more sense.